src/main/kotlin/calculator/domain/Calculator.kt

```diff
@@ -3,25 +3,18 @@ package calculator.domain
 class Calculator {
 
     fun calculate(numbers: List<String>): Int {
+        val numbers = convertStringToNumber(numbers)
         var sum = 0
         for (number in numbers) {
-            checkValidateNumber(number)
-            checkNegativeNumber(number)
-            sum += number.toInt()
+            sum += number.value
         }
 
         return sum
     }
 
-    private fun checkValidateNumber(string: String) {
-        try {
-            string.toInt()
-        } catch (e: NumberFormatException) {
-            throw RuntimeException("$string 은 올바른 숫자가 아닙니다.")
+    fun convertStringToNumber(numbers: List<String>): List<Number> {
```

Comment:
`Calculator` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun convertStringToNumber(numbers: List<String>): List<Number> {
```